Bridgecraft/Utils.swift

```diff
@@ -13,27 +13,43 @@ enum BridgecraftError: Error {
 }
 
 @discardableResult
-func shell(_ command: String, args: [String]) throws -> String {
+func shell(_ command: String, args: [String], verbose: Bool = false) throws -> String {
+    
     let ps = Process()
     ps.launchPath = command
     ps.arguments = args
     
-    let pipe = Pipe()
-    ps.standardOutput = pipe
+    let outputPipe = Pipe()
+    let errorPipe = Pipe()
+    ps.standardOutput = outputPipe
+    ps.standardError = errorPipe
     
     ps.launch()
     
-    var buffer = Data()
-    while ps.isRunning {
-        buffer.append(pipe.fileHandleForReading.readDataToEndOfFile())
+    ps.waitUntilExit()
+    
+    let outputPipeResult = outputPipe.fileHandleForReading.readDataToEndOfFile()
+    let errorPipeResult = errorPipe.fileHandleForReading.readDataToEndOfFile()
+    
+    guard let output = String(data: outputPipeResult, encoding: String.Encoding.utf8),
+    let error = String(data: errorPipeResult, encoding: String.Encoding.utf8) else {
```

Comment:
coud you please reformat this like:
```
guard
    xxx,
    yyy
else {
    zzz
}
```
?

Reply:
Done. The Xcode auto-formatting put it like this:
```
guard
    xxx,
    yyy
    else {
        zzz
}
```
It is ok? Or should I change it?